.changeset/fluffy-bats-tie.md

```diff
@@ -0,0 +1,11 @@
+---
+"@astrouxds/astro-web-components": patch
+"angular-workspace": patch
+"@astrouxds/angular": patch
+"astro-angular": patch
```

Comment:
@micahjones13 does it matter if we leave this in for release? the framework tests dont need to be included but Im not sure if it actually matters?

Reply:
I don't think so. I've been removing them since it hasn't been too cumbersome, but even if we left them in I'm p sure it wouldn't matter at all